### DISCUSSION TITLE
Assign habits to categories / Show by category

### DISCUSSION THREAD

**Discussion by bwilliams-uk** [iSoron/uhabits#1006] [Feature Requests]
For users who wish to track a large number of habits, it would be helpful to have a grouping feature that enables the user to view habits only belonging to a certain category. 

This could be achieved by enabling the user to create tabs above the habit list, which habits can then be dragged into. Tapping on the tab would only show habits belonging to that group. ​

This would make it easier to find habits without having to scroll far down the page. It also removes the visual distraction of other habits when only analysing a certain category and prevents accidental checking/unchecking of habits in other categories.

Similar functionality may be achieved by a "filter by colour" option, however the option to have colour-coded habits within each named category would offer a better experience.

**Comment by hiqua** [comment]
See https://github.com/iSoron/uhabits/discussions/495
